### PR TITLE
Release staging 20180829

### PR DESCRIPTION
**Pull request by rexwangcc**

### CODE REVIEW

**Comment by rexwangcc** [comment]
Waiting for Blue Box's inputs on changelogs, after which I will squash the commits.

File: components.yml [hunk 1]
```diff
@@ -10,20 +10,23 @@ deployed_component_versions:
...
-    greenbox-cromwell: 34-66c0fa6-SNAP,
-    cromwell_tools_version: v0.3.2,
-  data-store: staging-2018-07-11-16-08-27.release
+    greenbox-cromwell: 35-0a01d91-SNAP
+    greenbox-falcon: v0.1.2
+    greenbox-lira: v0.11.3
+    greenbox-pipeline-tools: v0.25.0
+    greenbox-skylab:
+      smartseq2: smartseq2_v1.0.0
+  data-store: integration-2018-08-15-05-55-50.release
```
Comment (Bento007) [inline components.yml]: should be `ddc0864d37697e753ba040341fc33bbab8e79494`

Comment (rexwangcc) [reply components.yml]: sure, will change

File: components.yml [hunk 2]
```diff
@@ -10,20 +10,23 @@ deployed_component_versions:
   # e.g.
   # ingest: v0.1.2
   upload: v2.2.2
-  metadata-schema: 4.4.0
+  metadata-schema: http://api.ingest.integration.data.humancellatlas.org/schemas/search/latestSchemas
```
[the remark this answers is not in the document]
Comment (rexwangcc) [reply components.yml]: Nice catch!

Comment (malloryfreeberg) [inline components.yml]: Wondering whether this should point to `api.ingest.staging` instead of `api.ingest.integration`?

Comment (rexwangcc) [reply components.yml]: fixed :)

**Pull request merged**